python/data_structures/linked_list.py

```diff
@@ -33,12 +33,57 @@ def includes(self, value):
             current = current.next
         return False
 
+    def append(self, new_value):
+        new_node = Node(new_value)
+        # if self.head is None:
+        #     self.head = new_node
+        #     return
+        # else:
+        saved_value = self.head
+        while saved_value.next is not None:
+            saved_value = saved_value.next
+        saved_value.next = new_node
+
+    def insert_before(self, value, new_value):
+        if self.head is None:
+            raise TargetError
+        if self.includes(value) is False:
+            raise TargetError
```

Comment:
While I understand the logic of having 2 separate conditions here, consider how `includes()` behaves when `self.head` is None. Since the result is the same, could lines 48 - 51 be condensed to 1 condition?